fecgov/openFEC
# Feature/6009 add case tests

## Summary (required)

- Resolves #6009 

This ticket adds integration testing with Elasticsearch to test all case filters. I will make separate PRs for AOs, statutes, and the other legal endpoints. 


### Required reviewers 2 - 3 backend developers


## Impacted areas of the application

General components of the application that this PR will affect:

- circleci pipeline
- pytest


## How to test

- checkout this branch
- start elasticsearch in new terminal 
- run `pytest` in another terminal
- to see output from any individual test uncomment the logging statements and run ` pytest tests/integration/test_cases_elasticsearch.py -s` 

Note: running locally will clear out your elasticsearch environment 
- look at codecov to see increase in coverage https://app.codecov.io/gh/fecgov/openFEC/blob/feature%2F6009-add-case-tests/webservices%2Fresources%2Flegal.py
